lib/mobility-core/src/Kernel/External/AadhaarVerification/Interface/Gridline.hs

```diff
@@ -43,7 +43,7 @@ generateAadhaarOtp cfg req = do
     AadhaarVerificationResp
       { statusCode = resp._data.code,
         message = resp._data.message,
-        transactionId = resp._data.transaction_id,
+        transactionId = fromMaybe " " resp._data.transaction_id,
```

Comment:
Isn't it better to pass it as Nothing and handle inside the Domain?

Reply:
so instead of " "  this should i pass nothing